src/app/user-lists/user-lists.component.html

```diff
@@ -1,4 +1,7 @@
-<ol *ngFor='let element of Utenti; index as i;'>
-    <!-- {{i+1}} <img src="{{element.profilePhotoUrl}}" alt="gatti"><strong> {{element.name}}{{element.surname}}</strong> {{element.age}} {{element.address.city}} -->
-</ol>
-<app-user-singles [box]="Utenti"></app-user-singles>
\ No newline at end of file
+<div *ngIf='Utenti.length > 0; else noUsers'>
```

Comment:
dovresti evitare di utilizzare 2 lingue all'interno di un progetto, in generale si usa l'inglese, e mai le maiuscole per le variabili